### ISSUE TITLE
Bug: use-baseline: `background-attachment` incorrectly reported as error

### ISSUE THREAD

**Issue by jimmy-zhening-luo** [eslint/css#398]
### Environment

ESLint version: latest
@eslint/css version: latest
Node version: latest
npm version: latest
Operating System: Windows

### Which language are you using?

stylesheet

### What did you do?

<details>
<summary>Configuration</summary>

```json
{
  "css/use-baseline": 2
}
```
</details>

```css
body {
  background-attachment: fixed;
}
```

### What did you expect to happen?

no error, as it's been widely available since 2015 (11 (eleven) years ago)

### What actually happened?

Value 'fixed' of property 'background-attachment' is not a widely available baseline feature.eslint[css/use-baseline](https://github.com/eslint/css/blob/main/docs/rules/use-baseline.md)
Specifies whether the background images are fixed with regard to the viewport ('fixed') or scroll along with the element ('scroll') or its contents ('local').

Baseline icon Widely available across major browsers (Baseline since 2015)

Syntax: <attachment>#

[MDN Reference](vscode-file://vscode-app/c:/Users/zluo/AppData/Local/Programs/Microsoft%20VS%20Code/ce099c1ed2/resources/app/out/vs/code/electron-browser/workbench/workbench.html)

### Link to Minimal Reproducible Example

https://example.com

### Participation

- [ ] I am willing to submit a pull request for this issue.

### AI acknowledgment

- [x] I did not use AI to generate this issue report.
- [ ] (If the above is not checked) I have reviewed the AI-generated content before submitting.

**Comment by DMartens** [comment]
Thank you for the report.
This is an upstream issue from [`web-features`](https://github.com/web-platform-dx/web-features/blob/d01dd9014d5eb5929922dab0a685443f09bcbdf4/features/background-attachment.yml).
The baseline was recently (end of January) reset to false due to [a bug in Safari](https://github.com/web-platform-dx/web-features/pull/3721/changes#r2741959690).
I'm going to close this issue as we will update the data should it change and users can temporarily ignore it with `allowPropertyValues`.
